docs/logs/field-extractor-processor.md

```diff
@@ -0,0 +1,29 @@
+title: Field Extractor Processor
+description: Extract fields using a GROK pattern
+
+Structuring data into fields is important if you are using Sematext Logs.  Having your logs structured makes it easy to analyze logs, create charts with data from individual fields, filter log events, or group them by their field values.
+
+By using Grok Patterns Extractor, you can extract multiple fields from other fields. With GROK patterns, you can efficiently extract essential information from lengthy text fields, eliminating redundant data, thus reducing costs. Check out [How Logs Pipelines Can Reduce Your Log Monitoring Costs](../logs/reduce-costs-with-pipelines) for additional information.
```

Comment:
Is it Grok Patterns Extractor or Field Extractor?

Reply:
fixed